microservice-schedule/src/main/java/io/microprofile/showcase/schedule/cdi/CacheProducer.java

```diff
@@ -6,40 +6,37 @@
 package io.microprofile.showcase.schedule.cdi;
 
 import io.microprofile.showcase.schedule.model.Schedule;
+import io.microprofile.showcase.schedule.persistence.LongKey;
 
 import javax.annotation.PostConstruct;
-import javax.annotation.PreDestroy;
 import javax.cache.Cache;
 import javax.cache.CacheManager;
-import javax.cache.Caching;
 import javax.cache.configuration.MutableConfiguration;
 import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
 
 /**
  * @author mike
  */
 @ApplicationScoped
 public class CacheProducer {
 
+  @Inject
```

Comment:
no need to create CacheManager - Payara can inject it. Injected CacheManager will use the same Hazelcast instance as the instance used by Payara for JCache CDI interceptors.